src/Pages/ADT3DScenePage/ADT3DScenePage.styles.ts

```diff
@@ -7,7 +7,7 @@ const classNames = {
     header: `${scenePageClassPrefix}-header`
 };
 
-export const getStyles = memoizeFunction(() => {
+export const getStyles = memoizeFunction((isBuildMode: boolean) => {
```

Comment:
Minor thing, but can we make it an enum for `mode: 'build'|'viewer'`? Just makes it easier to extend if for some reaosn we need more than 2 states.

Reply:
This actually needs to be a boolean, because we're doing multiple checks 
```ts
deeplinkState.mode === ADT3DScenePageModes.BuildScene && state.currentStep === ADT3DScenePageSteps.Scene
```
To get the `isBuildMode` param.  If we were just passing the mode through, I think this makes sense, but keeping as is for now.